scripts-dev/add_admin.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import json
+import sqlite3
+import logging
+
+
+def reg_admin(cursor):
+    headers = {'content-type': 'application/json'}
+    url = 'http://localhost:8008/_matrix/client/api/v1/register'
+    post_data = {'user': 'admin',
+                 'password': 'admin123',
+                 'type': 'm.login.password'}
```

Comment:
I'm a bit hesitant to add example scripts like this, as they can't directly be run. I believe the `register_new_user` script actually has an option to specify the new user is an admin or not.